editor/src/clj/editor/editor_extensions/ui_docs.clj

```diff
@@ -211,8 +237,7 @@
 
 (def ^:private heading-props
   (-> typography-specific-props
-      (conj (make-prop :level :coerce (coerce/enum 1 2 3 4 5 6) :types ["integer"]
-                       :doc "Heading level, an integer from 1 to 6, defaults to 3 (intended value for headings in dialog headers)"))
+      (conj (enum-prop :style :enum :heading-style :doc "heading style, defaults to <code>editor.ui.HEADING_STYLE.DIALOG</code>"))
```

Comment:
It is a bit weird that we're defaulting to `HEADING_STYLE.DIALOG`. Maybe we should change the default to something a bit more neutral like `HEADING_STYLE.H3`, seeing as how they are identical?

Reply:
Okay!